File: mysql_examples/recipes/apache/web_connect/web_connect.py
#!/usr/bin/python

import sys
import MySQLdb
import Cookbook

print """Content-Type: text/html

<html>
<head>
<title>Python Web Connect Page</title>
</head>
<body bgcolor="white">
"""

conn = Cookbook.connect ()
print "<p>Connected</p>"
conn.close ()
print "<p>Disconnected</p>"

print """
</body>
</html>
"""

File: mysql_examples/recipes/apache/web_connect/web_connect.php
<html>
<head>
<title>PHP Web Connect Page</title>
</head>
<body bgcolor="white">

<?php

require_once "Cookbook.php";

$conn =& Cookbook::connect ();
print ("<p>Connected</p>\n");
$conn->disconnect ();
print ("<p>Disconnected</p>\n");

?>
